toolkit/tools/imagecustomizer/docs/configuration.md

```diff
@@ -160,9 +162,11 @@ os:
         - [remove](#remove-string)
         - [updateLists](#updatelists-string)
         - [update](#update-string)
-    - [additionalFiles](#additionalfiles-mapstring-fileconfig)
-      - [fileConfig type](#fileconfig-type)
-        - [path](#fileconfig-path)
+    - [additionalFiles](#os-additionalfiles)
+      - [additionalFile type](#additionalfile-type)
```

Comment:
We should also update `additionalDirs` to follow same `source` `destination` terminology

Reply:
Ok, I've opened a work item.